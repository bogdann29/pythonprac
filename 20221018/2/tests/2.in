:pow x y pow(x,y)
:print x print(x)
pow 2 3
print pow(5,6)
pow 1 0
quit "{}, {}"
