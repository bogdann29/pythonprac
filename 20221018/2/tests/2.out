8.0
15625.0
None
1.0
3, 6
